specs/clinicalServices/frontDesk.cpt

```diff
@@ -126,3 +126,36 @@
 * wait for overlay to disappear
 * Save visit data
 * wait for overlay to disappear
+
+# Create new patient with random details and start an OPD visit
+* Open "Registration" module
+* Create a new patient with random patient detail
+* Start an OPD Visit
+
+# Create a new patient with random patient detail
```

Comment:
A similar concept with name "Create a new patient with random patient details" is present, what is the difference?

Reply:
"Create a new patient with random patient details" is suitable for bahmni lite registration page,in snomed ,registration page is different that is why I have created a new concept

Reply:
Can you rename the step accordingly? both the below steps looks same.
Create a new patient with random patient details
Create a new patient with random patient detail

Reply:
okayy

Reply:
done